src/vscode-cosmosdb.api.d.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export interface CosmosDBDatabase {
+    accountName: string
+    connectionString: string
+    treeItemId: string
+    databaseName: string | undefined
```

Comment:
databaseName should always be defined

Reply:
Agreed, but have a feeling that this API became very strict

Reply:
You either need to make databaseName always defined or change the name of the interface - because it doesn't make sense for something named `CosmosDBDatabase` not to have `databseName`